src/engraving/dom/chord.cpp

```diff
@@ -1886,9 +1890,9 @@ void Chord::undoChangeSpanArpeggio(Arpeggio* a)
         }
         Chord* chord = toChord(linkedObject);
         Score* score = chord->score();
-        EngravingItem* linkedArp = chord->spanArpeggio();
-        if (score && linkedArp) {
-            score->undo(new ChangeSpanArpeggio(chord, toArpeggio(linkedArp)));
+        Arpeggio* linkedArp = a ? static_cast<Arpeggio*>(a->linkedClone()) : nullptr;
```

Comment:
After another look, what I should have suggested was:
```suggestion
        Arpeggio* linkedArp = a ? static_cast<Arpeggio*>(a->findLinkedInScore(score)) : nullptr;
```
However, this causes the playback bug to reappear.

Reply:
What's the difference between these two?

Reply:
`a->linkedClone()` creates a new cloned element, however one should already exist in the part which we need to find with `a->findLinkedInScore(score)`.

Reply:
>however one should already exist in the part

We should not rely on the fact that it already exists. We should probably use a combination of findLinkedInScore() and linkedClone() here (I'm not sure this is a good idea though, as findLinkedInScore does a lot of things, and it might be quite inefficient here). Anyway, I've removed this piece of code and replaced it with a TODO (as it did nothing). Please fix it in a separate PR. I'm also going to completely fix the playback issue in a separate PR, as it requires significant changes and port only this PR to 4.3

Reply:
I'll take a look at it, thanks for sorting the playback out.